load.php

```diff
@@ -260,3 +270,53 @@ function perflab_load_active_and_valid_modules() {
 if ( is_admin() ) {
 	require_once PERFLAB_PLUGIN_DIR_PATH . 'admin/load.php';
 }
+
+/**
+ * Trigger actions when a module gets activated or deactivated.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $option    The option name.
+ * @param mixed  $old_value Old value of the option.
+ * @param mixed  $value     New value of the option.
+ *
+ * @return void
+ */
+function perflab_run_module_activation_deactivation( $option, $old_value, $value ) {
+	if ( PERFLAB_MODULES_SETTING !== $option ) {
+		return;
+	}
```

Comment:
See my other comment, by using a more targeted action we can remove this.

Reply:
fixed